modular_chomp/code/modules/mob/living/carbon/human/species/station/protean/protean_powers.dm

```diff
@@ -705,3 +705,45 @@
 	to_call = /mob/living/carbon/human/proc/nano_copy_body
 
 #undef PER_LIMB_STEEL_COST
+
+/mob/living/carbon/human/proc/chest_transparency_toggle()
+	set name = "transparency toggle (chest only)"
+	set category = "Abilities.Protean"
+	if(stat || world.time < last_special)
+		return
+	last_special = world.time + 50
+	var/temporgans = src.organs
```

Comment:
not really needed, the src.organs or just organs can go directly into the for loop

Reply:
fixed